test/support/utils.h

```diff
@@ -636,6 +636,17 @@ transform_reduce_serial(InputIterator first, InputIterator last, T init, BinaryO
     return init;
 }
 
+template <typename TState>
+void
+exitOnError(TState state, const char* sFailMsg = "Fail")
```

Comment:
Should this name be in snake case (i.e., `exit_on_error`) to match the rest of the functions in this file?

Reply:
Fixed: renamed to ``exit_on_error```